Sources/CombineCoreData/PerformPublisher.swift

```diff
@@ -0,0 +1,106 @@
+import Combine
+import CoreData
+
+/// A publisher that asynchronously performs a given `block` on the context’s queue.
+public struct PerformPublisher<Output>: Publisher {
```

Comment:
Why can't we make it an internal structure? Anyway you will `eraseToAnyPublisher` every time in your application.

Reply:
I looked at `Combine` and `Alamofire` where the types of publishers are not hidden. 
I didn't find a recommendation for third party libraries to use `eraseToAnyPublisher`.
I think `eraseToAnyPublisher` is used to hide composite publishers in the app `Just(1).map { String($0) }. eraseToAnyPublisher()`

Reply:
asked on the forum 🍏
https://developer.apple.com/forums/thread/652373